test/e2e/15_upgrade.bats

```diff
@@ -74,8 +75,43 @@ function setup() {
   git push >&3
 
   # Assert change is rolled out
-  poll_until_equals 'podinfo-git HelmRelease chart update' "successfully cloned chart revision: $head_hash" "kubectl -n $DEMO_NAMESPACE get helmrelease/podinfo-git -o jsonpath='{.status.conditions[?(@.type==\"ChartFetched\")].message}'"
-  poll_until_equals 'podinfo-git HelmRelease revision matches' "$head_hash" "kubectl -n $DEMO_NAMESPACE get helmrelease/podinfo-git -o jsonpath='{.status.revision}'"
+  poll_until_equals 'release chart update' "successfully cloned chart revision: $head_hash" "kubectl -n $DEMO_NAMESPACE get helmrelease/podinfo-git -o jsonpath='{.status.conditions[?(@.type==\"ChartFetched\")].message}'"
+  poll_until_equals 'revision match' "$head_hash" "kubectl -n $DEMO_NAMESPACE get helmrelease/podinfo-git -o jsonpath='{.status.revision}'"
+}
+
+@test "When a HelmRelease is nested in a chart, an upgrade does succeed" {
+  # Install chartmuseum
```

Comment:
@stefansedich this will probably help you to address the CRD test with a local dummy chart.

Reply:
Great think that will, will update that test and send a PR over.